text/0043-phrase-query.md

```diff
@@ -0,0 +1,161 @@
+- Title: Phrase Query
+- Start Date: 2021-06-02
+- Specification PR: [#43](https://github.com/meilisearch/specifications/pull/43)
+- MeiliSearch Tracking-Issues:
+
+# Phrase Query
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Meilisearch does not allow users a way to write a strict query in order to ask the engine to be more drastic in its selection of candidates for search results. The Phrase Query feature adds a simple syntax available to users to require the engine to select documents that will strictly match the search field value. That is, without typography, without prefix and, in the order of the given terms.
+
+### II. Motivation
+
+This feature is driven by user needs. Indeed, let's take an example recently brought up in our community slack.
+
+The user in question would like to be able to retrieve specifically the document containing the unique ISBN identifier and only that one. In an ux context of type as you search, this is impossible today without impacting the UI/UX or finding a workaround.
+
+- It could have had a specific search field only dedicated to this ISBN field in order to make it a filter and using the `=` operator. Not great for the ux.
```

Comment:
```suggestion
- It could have had a specific search field only dedicated to this ISBN field in order to make it a filter and using the `=` operator. Not great for the UX.
```